AIX build break because of recent changes

OpenBLAS build in AIX fails now with the following error., 

/opt/IBM/openxlC/17.1.2/bin/ibm-clang_r -c -O2 -DSMALL_MATRIX_OPT -DGEMM_GEMV_FORWARD -DGEMM_GEMV_FORWARD_BF16 -DMAX_STACK_ALLOC=2048 -fopenmp -DHAVE_P10_SUPPORT -Wall -DF_INTERFACE_IBM -fPIC -DDYNAMIC_ARCH -DSMP_SERVER -DUSE_OPENMP -DNO_WARMUP -DMAX_CPU_NUMBER=16 -DMAX_PARALLEL_NUMBER=1 -DBUILD_BFLOAT16 -DBUILD_SINGLE=1 -DBUILD_DOUBLE=1 -DBUILD_COMPLEX=1 -DBUILD_COMPLEX16=1 -DVERSION=\"0.3.29.dev\" -DUSE_OPENMP -fopenmp -fno-integrated-as -m64 -UASMNAME -UASMFNAME -UNAME -UCNAME -UCHAR_NAME -UCHAR_CNAME -DASMNAME=setparam_POWER6 -DASMFNAME=setparam_POWER6_ -DNAME=setparam_POWER6_ -DCNAME=setparam_POWER6 -DCHAR_NAME=\"setparam_POWER6_\" -DCHAR_CNAME=\"setparam_POWER6\" -DNO_AFFINITY -DTS=_POWER6 -I.. -DBUILD_KERNEL -DTABLE_NAME=gotoblas_POWER6 -UDOUBLE  -UCOMPLEX setparam_POWER6.c -o setparam_POWER6.o

**setparam_POWER6.c:77:20: error: incompatible function pointer types initializing** 'int (*)(BLASLONG, BLASLONG, BLASLONG, float, float *, BLASLONG, float *, BLASLONG, float *, BLASLONG)' (aka 'int (*)(long, long, long, float, float *, long, float *, long, float *, long)') with an expression of type 'int (BLASLONG, float *, BLASLONG, float *, BLASLONG, float *)' (aka 'int (long, float *, long, float *, long, float *)') [-Wincompatible-function-pointer-types]
   77 |   srot_k_POWER6,   srotm_k_POWER6,  saxpy_k_POWER6,  sscal_k_POWER6, sswap_k_POWER6,
` `|`              ` ^~~~~~~~~~~~~~
setparam_POWER6.c:77:70: error: incompatible function pointer types initializing 'int (*)(BLASLONG, BLASLONG, float, bfloat16 *, BLASLONG, bfloat16 *, BLASLONG, float, float *, BLASLONG)' (aka 'int (*)(long, long, float, unsigned short *, long, unsigned short *, long, float, float *, long)') with an expression of type 'int (BLASLONG, BLASLONG, BLASLONG, float, float *, BLASLONG, float *, BLASLONG, float *, BLASLONG)' (aka 'int (long, long, long, float, float *, long, float *, long, float *, long)') [-Wincompatible-function-pointer-types]
   77 |   srot_k_POWER6,   srotm_k_POWER6,  saxpy_k_POWER6,  sscal_k_POWER6, sswap_k_POWER6,
`  `|`                                                                   `^~~~~~~~~~~~~~

Looks like this is because of the changes went in #5069

--- a/common_param.h
+++ b/common_param.h
@@ -77,6 +77,7 @@ BLASLONG (*isbmin_k) (BLASLONG, float *, BLASLONG);
   double (*dsbdot_k) (BLASLONG, float *, BLASLONG, float *, BLASLONG);
 
   int    (*sbrot_k)  (BLASLONG, float *, BLASLONG, float *, BLASLONG, float, float);
+  int    (*sbrotm_k) (BLASLONG, float *, BLASLONG, float *, BLASLONG, float *);
 
   int    (*sbaxpy_k) (BLASLONG, BLASLONG, BLASLONG, float, float *, BLASLONG, float *, BLASLONG, float *, BLASLONG);
   int    (*sbscal_k) (BLASLONG, BLASLONG, BLASLONG, float, float *, BLASLONG, float *, BLASLONG, float *, BLASLONG);

